LightweightCharts.podspec

```diff
@@ -19,9 +19,9 @@ Pod::Spec.new do |s|
       :script => 'LIBRARY=${TEMP_DIR}/lightweight-charts.js
       
 if [ "${CONFIGURATION}" = "Release" ]; then
-    curl -o $LIBRARY https://unpkg.com/lightweight-charts@1.2.1/dist/lightweight-charts.standalone.production.js
+    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0/dist/lightweight-charts.standalone.production.js
```

Comment:
```suggestion
    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0.0/dist/lightweight-charts.standalone.production.js
```

Reply:
Can we use `s.version` here as variable? Or maybe define it once and use in all places?

Reply:
This script copy to Xcode target just as plain text and we can not specify variables from other scope to it